miden-lib/asm/miden/tx.masm

```diff
@@ -195,7 +195,21 @@ export.execute_foreign_procedure
     # reset the current account data offset to the native offset (2048)
     exec.kernel_proc_offsets::end_foreign_context_offset
     # => [offset, <values returned from the foreign procedure>]
-    
+
     syscall.exec_kernel_proc
     # => [<values returned from the foreign procedure>]
 end
+
+export.update_expiration_delta
```

Comment:
Let's add doc comments here (I think they can probably just copied from the kernel procedure).